had just downloaded a project that caused merge...

Notes
======
had just downloaded a project that caused merge conflicts

Environment
======
Environment Key | Value
:----: | :----:
version | 11.0
build | 143
UDID | 3c7d0770c6265875
Android Release | 5.1.1
Android SDK | 22
Brand | google
Device | flo
Stack trace
======
```java
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.translationstudio.androidapp/com.door43.translationstudio.ui.translate.TargetTranslationActivity}: java.lang.NullPointerException: Attempt to read from field 'java.lang.String org.unfoldingword.resourcecontainer.Project.languageSlug' on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2325)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2387)
	at android.app.ActivityThread.access$800(ActivityThread.java:151)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1303)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:135)
	at android.app.ActivityThread.main(ActivityThread.java:5254)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
Caused by: java.lang.NullPointerException: Attempt to read from field 'java.lang.String org.unfoldingword.resourcecontainer.Project.languageSlug' on a null object reference
	at com.door43.translationstudio.ui.translate.FirstTabFragment.onCreateView(FirstTabFragment.java:59)
	at android.app.Fragment.performCreateView(Fragment.java:2053)
	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:894)
	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067)
	at android.app.BackStackRecord.run(BackStackRecord.java:834)
	at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1452)
	at android.app.Activity.performStart(Activity.java:6005)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2288)
	... 10 more

```
Log history
======
```java
11/10/16 10:47 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:47 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation en-x-demo2_php_text_reg
11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation en-x-demo2_php_text_reg
11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:46 AM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation kcg_mrk_text_reg
11/10/16 10:46 AM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en-x-demo2_obs_text_obs
11/10/16 10:46 AM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en-x-demo1_luk_text_reg
11/10/16 10:46 AM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en-x-demo1_rut_text_reg
11/10/16 10:46 AM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en-x-demo2_tit_text_reg
11/10/16 10:46 AM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en-x-demo1_tit_text_reg
11/10/16 10:46 AM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en_3jn_text_reg
11/10/16 10:46 AM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en-x-demo1_3jn_text_reg
11/10/16 10:46 AM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en_2jn_text_reg
11/10/16 10:46 AM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en-x-demo1_jud_text_reg
11/10/16 10:46 AM  E/com.door43.translationstudio.tasks.UpdateAppTask: Failed to commit migration changes to target translation en-x-demo1_phm_text_reg
11/10/16 10:46 AM  E/com.door43.translationstudio.core.TargetTranslation: Failed to commit changes
org.eclipse.jgit.api.errors.ConcurrentRefUpdateException: Could not lock HEAD. RefUpdate return code was: LOCK_FAILURE
	at org.eclipse.jgit.api.CommitCommand.call(CommitCommand.java:300)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:991)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:943)
	at com.door43.translationstudio.tasks.UpdateAppTask.updateTargetTranslations(UpdateAppTask.java:143)
	at com.door43.translationstudio.tasks.UpdateAppTask.start(UpdateAppTask.java:78)
	at org.unfoldingword.tools.taskmanager.ManagedTask.run(ManagedTask.java:78)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)

11/10/16 10:46 AM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation aaa_php_text_ulb
11/10/16 10:46 AM  W/com.door43.translationstudio.tasks.UpdateAppTask: Failed to migrate the target translation aaa_php_text_ulb
     

```

--- a/app/src/main/java/com/door43/translationstudio/ui/translate/FirstTabFragment.java
+++ b/app/src/main/java/com/door43/translationstudio/ui/translate/FirstTabFragment.java
@@ -55,17 +55,16 @@ public class FirstTabFragment extends BaseFragment implements ChooseSourceTransl
         ImageButton newTabButton = (ImageButton) rootView.findViewById(R.id.newTabButton);
         LinearLayout secondaryNewTabButton = (LinearLayout) rootView.findViewById(R.id.secondaryNewTabButton);
         TextView translationTitle = (TextView) rootView.findViewById(R.id.source_translation_title);
-        Project p = mLibrary.index().getProject(App.getDeviceLanguageCode(), targetTranslation.getProjectId(), true);
-        List<Resource> resources = mLibrary.index().getResources(p.languageSlug, p.slug);
-        ResourceContainer  resourceContainer = null;
         try {
-            resourceContainer = mLibrary.open(p.languageSlug, p.slug, resources.get(0).slug);
+            Project p = mLibrary.index().getProject(App.getDeviceLanguageCode(), targetTranslation.getProjectId(), true);
+            List<Resource> resources = mLibrary.index().getResources(p.languageSlug, p.slug);
+            ResourceContainer  resourceContainer = mLibrary.open(p.languageSlug, p.slug, resources.get(0).slug);
+//        SourceLanguage sourceLanguage = mLibrary.getPreferredSourceLanguage(targetTranslation.getProjectId(), App.getDeviceLanguageCode());
+            translationTitle.setText(resourceContainer.readChunk("front", "title") + " - " + targetTranslation.getTargetLanguageName());
         } catch (Exception e) {
-            e.printStackTrace();
+            Logger.e(FirstTabFragment.class.getSimpleName(),"Error getting resource container for '"+ targetTranslationId + "' and project '" + targetTranslation.getProjectId() + "'",e);
         }
 
-//        SourceLanguage sourceLanguage = mLibrary.getPreferredSourceLanguage(targetTranslation.getProjectId(), App.getDeviceLanguageCode());
-        translationTitle.setText(resourceContainer.readChunk("front", "title") + " - " + targetTranslation.getTargetLanguageName());
 
         View.OnClickListener clickListener = new View.OnClickListener() {
             @Override

